cmd/environment.go

```diff
@@ -120,6 +120,7 @@ func newEnvironmentCreateCmd() *cobra.Command {
 	flags.StringVarP(&provider, "provider", "p", "quorum", "underlying protocol to use for this network, quorum or geth")
 	flags.StringVarP(&consensus, "consensus", "k", "raft", "consensus algorithm to use for the given protocol, raft or ibft for quorum, poa for geth")
 	flags.BoolVarP(&multiRegion, "multi-region", "R", false, "whether to enable multi region")
+	flags.IntVarP(&blockPeriod, "block-period", "P", 10, "block period in seconds")
```

Comment:
The default is different for different providers.

So I think we need to have `0` as the default on the CLI parameter here, which is the same on the API as omitting the field - allowing the Kaleido platform to set the appropriate default.